op-test

```diff
@@ -106,6 +106,8 @@ from testcases import OpTestDlparIO
 from testcases import OpTestLPM
 from testcases import OpTestKexec
 from testcases import OpTestGSBStaticKey
+from testcases import gcov_setup
```

Comment:
please import GcovSetup and have same name for test py file as well

Reply:
done